app/helpers/goods_nomenclature_helper.rb

```diff
@@ -8,7 +8,7 @@ def goods_nomenclature_path(path_opts = {})
 
     case current_goods_nomenclature_code&.size
     when nil
-      sections_path(path_opts)
+      find_commodity_path(path_opts)
```

Comment:
Not sure on this - should this one be going to `browse_sections_path`?

Reply:
So the fallback path here should be the home page, yeah, so this is consistent with previous behaviour.

Reply:
Unfortunately we can't use root_path